python/ray/data/impl/compute.py

```diff
@@ -121,13 +122,24 @@ def apply(
         return BlockList(list(new_blocks), list(new_metadata))
 
 
+@PublicAPI
 class ActorPool(ComputeStrategy):
```

Comment:
nit: Not sure if it's just me but I feel ActorPool[Compute]Strategy is a better name.

ActorPool makes me feel that I'm having a pool of actors that can be reused (i.e. actors are class member)
```
pool = ActorPool(min, max)
ds.map_batch(pool, num_gpus=1)
ds.map_batch(pool, num_gpus=2) // Intuitively this doesn't seem right since they have incompatible num_gpus
```
but
```
strategy = ActorPoolStrategy(min, max)
ds.map_batch(strategy, num_gpus=1)
ds.map_batch(strategy, num_gpus=2)
```
is clear that I'm only defining a strategy and the actors themself are not accessible and reusable.

Reply:
Good point. Renamed.